book/src/writing-apps/solidity.md

```diff
@@ -0,0 +1,52 @@
+# Solidity SDK
+
+As a supplement to OpenVM, we provide a [Solidity SDK](https://github.com/openvm-org/openvm-solidity-sdk) containing OpenVM verifier contracts generated at each official release commit using the `cargo openvm setup` command. Note that these builds are for the default aggregation VM config which should be sufficient for most users. If you use a custom config, you will need to manually generate the verifier contract using the [OpenVM SDK](../advanced-usage/sdk.md).
+
+## Installation
+
+To install the Solidity SDK as a dependency into your forge project, run the following command:
+
+```bash
+forge install openvm-org/openvm-solidity-sdk
+```
+
+## Usage
+
+Once you have the SDK installed, you can import the SDK contracts into your solidity project:
+
+```solidity
+import "openvm-solidity-sdk/v1.1.1/OpenVmHalo2Verifier.sol";
+```
+
+If you are using an already-deployed verifier contract, you can simply import the `IOpenVmHalo2Verifier` interface:
+
+```solidity
+import { IOpenVmHalo2Verifier } from "openvm-solidity-sdk/v1.1.1/interfaces/IOpenVmHalo2Verifier.sol";
+
+contract MyContract {
+    function myFunction() public view {
+        // ... snip ...
+
+        IOpenVmHalo2Verifier(verifierAddress)
+            .verify(publicValues, proofData, appExeCommit, appVmCommit);
+
+        // ... snip ...
+    }
+}
+```
+
+The arguments to the `verify` function are the fields in the [EVM Proof JSON Format](./verify.md#evm-proof-json-format).
```

Comment:
either here or below, mention that if using the solidity-sdk's OpenVmHalo2Verifier, then the number of user public values is fixed to 32 bytes